### ISSUE TITLE
Other player from DB

### ISSUE THREAD

**Issue by GregLoichot** [tkrebs/ep3-bs#312]
Why partners field does not come from database ? It’s basically a free text input. 
Why not adding a option to allow that process ?

Best,
Greg

**Comment by cbits68** [comment]
Technical it should be possible but I think it would be difficult, because you provide privat data to other people which is against data protection (DSGVO). This has to be allowed by the user which provide personal to you whil accepting the terms and conditions of your data protection agreement.
If the user provide the name of the second player, he is responsible, that this person with it.

**Comment by GregLoichot** [comment]
Yep, but it should be available as an option.

**Comment by TCRW** [comment]
I agree, i would support this, too. Every member has to agree the DSGVO, an we have a passage, that the publication of names  is agreed to guarantee the whole club process.

**Comment by GregLoichot** [comment]
Yes would be really nice. And if the partners could receive the email when the booking is confirmed … amazing.
